### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by tom-borcin**
Adding issue templates into repository. Please check the templates and let me know if the provided documentation, description, requested data or whole templates make sense.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      options:
+        - label: I have read the documentation for [esp-protocols components](https://espressif.github.io/esp-protocols/) and the issue is not addressed there.
+          required: true
+        - label: I have updated my esp-protocols branch (master or release) to the latest version and checked that the issue is present there.
+          required: true
+        - label: I have searched the issue tracker for a similar issue and not found a similar issue.
+          required: true
+  - type: dropdown
+    id: component
+    attributes:
```
Comment (david-cermak) [inline .github/ISSUE_TEMPLATE/bug-report.yml:19]: Could we also ask for the actual component version?

Something like?
```suggestion
    attributes:
      label: component version
      description: Which component version does this issue occur on? (see your `idf_component.yml`)
      placeholder: 1.0.0
```

**Review by david-cermak — APPROVED** [review]
Thanks Tomas for helping us with the issue template!

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (tom-borcin) [reply .github/ISSUE_TEMPLATE/bug-report.yml:19]: Done

**Review by david-cermak — APPROVED** [review]

**Comment by david-cermak** [comment]
Thanks @tom-borcin for adding the template, this is very useful!

**Pull request merged**